solarforecastarbiter/reference_forecasts/tests/test_persistence.py

```diff
@@ -0,0 +1,288 @@
+from contextlib import nullcontext as does_not_raise
+from functools import partial
+
+import pandas as pd
+from pandas.util.testing import assert_series_equal
+
+from solarforecastarbiter import datamodel
+from solarforecastarbiter.reference_forecasts import persistence
+from solarforecastarbiter.conftest import default_observation
+
+import pytest
+
+
+def load_data_base(data, observation, data_start, data_end):
```

Comment:
@alorenzo175 does the actual API or `api.py` account for the interval labeling like this test function does? It looks like no. I think they should.

Reply:
The actual API does not and I don't think it should. You ask for values between a range and you get that range inclusive of the endpoints. No unexpected timestamps.

I guess it's ok for api.py

Reply:
Let's discuss before we do anything here.

Reply:
https://github.com/SolarArbiter/solarforecastarbiter-core/issues/80